docs/user_guide/use_case.md

```diff
@@ -40,7 +40,7 @@ The following will be included in this article：
 
 Before configuring yunikorn-config, we need to create users using [Authenticating](https://kubernetes.io/docs/reference/access-authn-authz/authentication/) and [RBAC](https://kubernetes.io/docs/reference/access-authn-authz/rbac/) from Kubernetes.
```

Comment:
This should be "Authentication" rather than "Authenticating" (grammar). Will fix on merge.